instrumentation-api-incubator/src/main/java/io/opentelemetry/instrumentation/api/incubator/builder/internal/DefaultHttpClientInstrumenterBuilder.java

```diff
@@ -224,6 +224,10 @@ InstrumenterBuilder<BUILDERREQUEST, BUILDERRESPONSE> instrumenterBuilder(
     return Instrumenter.builder(openTelemetry, instrumentationName, spanNameExtractor);
   }
 
+  public String getInstrumentationName() {
```

Comment:
this method does not seem to be used anywhere

Reply:
removed